lib/maplibre-compose/src/commonMain/kotlin/dev/sargunv/maplibrecompose/compose/CameraState.kt

```diff
@@ -50,6 +51,10 @@ public class CameraState internal constructor(firstPosition: CameraPosition) {
     map.animateCameraPosition(finalPosition, duration)
   }
 
+  public fun screenLocationFromPosition(position: Position): Offset? {
+    return map?.screenLocationFromPosition(position)
+  }
```

Comment:
```suggestion
  public fun screenLocationFromPosition(position: Position): Offset {
    return map?.screenLocationFromPosition(position) ?: error("Map requested before it was initialized")
  }
```

generally if this is called before the map is attached, that should be considered an error

maybe there's a use case for the caller to fail gracefully there, but I'd expose that as a separate `isInitialized` or something